node/pkg/raft/raft.go

```diff
@@ -10,11 +10,13 @@ import (
 	"github.com/rs/zerolog/log"
 
 	errorSentinel "bisonai.com/orakl/node/pkg/error"
+	"bisonai.com/orakl/node/pkg/utils/pool"
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
 	"github.com/libp2p/go-libp2p/core/host"
 )
 
 const HEARTBEAT_TIMEOUT = 100 * time.Millisecond
+const POOL_WORKER_COUNT = 3
```

Comment:
can you make this `POOL_WORKER_COUNT` configurable when initializing raft object? I don't think raft in reporter need multiple workers, but only aggregator

Reply:
lets take `workers` param from NewRaftNode and make it 1 from reporter, 3 from aggregator
```
func NewRaftNode(
	h host.Host,
	ps *pubsub.PubSub,
	topic *pubsub.Topic,
	messageBuffer int,
	leaderJobTimeout time.Duration,
) *Raft {
	r := &Raft{
		Host:  h,
		Ps:    ps,
		Topic: topic,

		Role:          "follower",
		VotedFor:      "",
		LeaderID:      "",
		VotesReceived: 0,
		Term:          0,
		Mutex:         sync.Mutex{},

		MessageBuffer:    make(chan Message, messageBuffer),
		Resign:           make(chan interface{}),
		HeartbeatTimeout: HEARTBEAT_TIMEOUT,

		LeaderJobTimeout: leaderJobTimeout,
	}
	return r
}
```

Reply:
done!